cli/Sources/Noora/Localization.swift

```diff
@@ -0,0 +1,148 @@
+    ///   - infoAlertTitle: The title to display in informational alerts.
+    ///   - infoAlertRecommendedTitle: The recommended title for informational alerts.
+    ///   - choicePromptFilterTitle: The title for the filter used in choice prompts.
+    ///   - choicePromptInstructionWithoutFilter: The instruction text for choice prompts when no filter is applied.
+    ///   - choicePromptInstructionWithFilter: The instruction text for choice prompts when filtering is enabled.
+    ///   - choicePromptInstructionIsFiltering: The instruction text for choice prompts while filtering is active.
+    ///   - textPromptValidationErrorsTitle: The title to display for validation errors in text prompts.
+    ///   - yesOrNoChoicePromptInstruction: The instruction text for yes/no choice prompts.
+    ///   - yesOrNoChoicePromptPositiveText: The localization for the positive answer option.
+    ///   - yesOrNoChoicePromptNegativeText: The localization for the negative answer option.
+    public init(
+        errorAlertTitle: String,
+        errorAlertRecommendedTitle: String,
+        warningAlertTitle: String,
+        warningAlertRecommendedTitle: String,
+        successAlertTitle: String,
+        successAlertRecommendedTitle: String,
+        infoAlertTitle: String,
+        infoAlertRecommendedTitle: String,
+        choicePromptFilterTitle: String,
+        choicePromptInstructionWithoutFilter: String,
+        choicePromptInstructionWithFilter: String,
+        choicePromptInstructionIsFiltering: String,
+        textPromptValidationErrorsTitle: String,
+        yesOrNoChoicePromptInstruction: String,
+        yesOrNoChoicePromptPositiveText: YesNoAnswerLocalization,
+        yesOrNoChoicePromptNegativeText: YesNoAnswerLocalization
+    ) {
+        self.errorAlertTitle = errorAlertTitle
+        self.errorAlertRecommendedTitle = errorAlertRecommendedTitle
+        self.warningAlertTitle = warningAlertTitle
+        self.warningAlertRecommendedTitle = warningAlertRecommendedTitle
+        self.successAlertTitle = successAlertTitle
+        self.successAlertRecommendedTitle = successAlertRecommendedTitle
+        self.infoAlertTitle = infoAlertTitle
+        self.infoAlertRecommendedTitle = infoAlertRecommendedTitle
+        self.choicePromptFilterTitle = choicePromptFilterTitle
+        self.choicePromptInstructionWithoutFilter = choicePromptInstructionWithoutFilter
+        self.choicePromptInstructionWithFilter = choicePromptInstructionWithFilter
+        self.choicePromptInstructionIsFiltering = choicePromptInstructionIsFiltering
+        self.textPromptValidationErrorsTitle = textPromptValidationErrorsTitle
+        self.yesOrNoChoicePromptInstruction = yesOrNoChoicePromptInstruction
+        self.yesOrNoChoicePromptPositiveText = yesOrNoChoicePromptPositiveText
+        self.yesOrNoChoicePromptNegativeText = yesOrNoChoicePromptNegativeText
+    }
+}
+
+extension Localization {
```

Comment:
I'd avoid using extensions as a scoping tool because it adds complexity to the compiler.